src/utils/client.js

```diff
@@ -49,6 +49,9 @@ async function connectToServer(url) {
  * @param {string} url - the base url of the server to connect to
  */
 async function getAccessToken(url) {
+  if (!process.env.REQUIRE_AUTH_FOR_OUTGOING || process.env.REQUIRE_AUTH_FOR_OUTGOING === 'false')
+    return '';
```

Comment:
Just out of curiosity (as an open to question to anyone on the team reading this): the app will still send an Authorization header just as `Authorization: Bearer `. That shouldn't cause any problems right? I know on our servers we won't even look at the Authorization header but could it be an issue else where?

Reply:
yea I started to remove the headers from all the requests but realized just sending the `Authorization: Bearer ` would be easier since all the requests call `getAccessToken`. I didn't see any problems while testing with our servers. I would imagine it should be fine with other servers too. If the server were to check the token it would just extract an empty string.

Reply:
It's impossible to say what arbitrary servers might do, but since the goal here is to disable the authorization process altogether I think it's safe to assume the target endpoint won't be looking at the Authorization header at all